src/components/WheelPicker/index.tsx

```diff
@@ -335,6 +335,8 @@ const WheelPicker = ({
     );
   }, []);
 
+  const offsets = items.map((_, i) => i * itemHeight);
```

Comment:
@Inbal-Tish 
Shouldn't we memoize this? 
This will generate a new list each render and might affect performance

Reply:
Yes, sure